### DISCUSSION TITLE
For example: min-[1620px] is not workin, tailwind dont see this, but media xl, 2xl, md he is see)) (sorry for my english)

### DISCUSSION THREAD

**Discussion by Evgeny-org** [tailwindlabs/tailwindcss#13758] [Help]
I used ti CDN 3.4.3 and use this class:

 <div class="
                    map-grayscale 
                    relative                                            xl:absolute 
                                                lg:translate-y-[-100px] xl:translate-y-0
                    left-0      md:left-[10%]   lg:left-[50%]                               2xl:left-[30%]  min-[1620px]:left-[20%]
                    top-0                                               xl:-top-1/4 
                    w-full      md:w-[80%]      lg:w-1/2                xl:w-2/3                            min-[1620px]:w-full
                                                lg:z-20                 xl:z-10
                    shadow-2xl
                    mb-16
">

TailWind dont see min-[1620px]:left-[20%] & min-[1620px]:w-full ((((((
                    
                    
![Uploading Снимок экрана 2024-05-28 в 23.20.15.png…]()
![Uploading Снимок экрана 2024-05-28 в 23.20.09.png…]()

**Comment by Nilomiranda** [comment]
@Evgeny-org you forgot to upload your screenshots properly.
